changelog/Microsoft.AAD.Reporting.json

```
@@ -1,6 +1,216 @@
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "Property",
+          "ChangedApiName": "incomingTokenType",
+          "ChangeType": "Addition",
+          "Description": "Added the **incomingTokenType** property to [signIn](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource.",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "String",
+          "ChangedApiName": "isTenantRestricted",
+          "ChangeType": "Deletion",
+          "Description": "Removed the **isTenantRestricted** resource type",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "String",
+          "ChangedApiName": "isTenantRestricted",
+          "ChangeType": "Addition",
+          "Description": "Added the [isTenantRestricted](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "String",
+          "ChangedApiName": "resourceTenantId",
+          "ChangeType": "Deletion",
+          "Description": "Removed the **resourceTenantId** resource type",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "String",
+          "ChangedApiName": "resourceTenantId",
+          "ChangeType": "Addition",
+          "Description": "Added the [resourceTenantId](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "Property",
+          "ChangedApiName": "riskEventTypes",
+          "ChangeType": "Deletion",
+          "Description": "Removed the **riskEventTypes** property from [signIn](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource.",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "String",
+          "ChangedApiName": "tokenIssuerType",
+          "ChangeType": "Deletion",
+          "Description": "Removed the **tokenIssuerType** resource type",
+          "Target": "signIn"
+        },
+        {
+          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
+          "ApiChange": "String",
+          "ChangedApiName": "tokenIssuerType",
+          "ChangeType": "Addition",
+          "Description": "Added the [tokenIssuerType](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource",
+          "Target": "signIn"
+        },
```

Conflict and wrong entries?
```suggestion
        {
          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
          "ApiChange": "String",
          "ChangedApiName": "tokenIssuerType",
          "ChangeType": "Deletion",
          "Description": "Removed the **tokenIssuerType** resource type",
          "Target": "signIn"
        },
        {
          "Id": "bd257a22-af29-466e-9b3d-1386183e61bf",
          "ApiChange": "Property",
          "ChangedApiName": "tokenIssuerType",
          "ChangeType": "Addition",
          "Description": "Added the **tokenIssuerType** property to the [signIn](https://docs.microsoft.com/en-us/graph/api/resources/signIn?view=graph-rest-beta) resource.",
          "Target": "signIn"
        },
```